File: 2DVirtualWorldSimulatorCpp/2DVirtualWorldSimulator/Wolf.cpp
#include <iostream>
#include "Wolf.h"

Wolf::Wolf() {
	this->name = 'W';
	this->strength = 9;
	this->initiative = 5;
}

char Wolf::GetName() const{
	return 'W';
}

string Wolf::GetOrganismFullName() const {
	return "Wolf";
}

Wolf::~Wolf() {

}

File: 2D_sym_python/animals/Wolf.py
from Animal import*

class Wolf(Animal):

    def __init__(self):
        super().__init__()
        self.name = 'W'
        self.strength = 9
        self.initiative = 5
    
    def GetName(self):
        return 'W'
    
    def GetOrganismFullName(self):
        return "Wolf"